src/Akeneo/Connectivity/Connection/front/src/connect/components/ConnectedApps/ConnectedAppsContainer.tsx

```diff
@@ -41,6 +41,7 @@ export const ConnectedAppsContainer: FC<Props> = ({allConnectedApps}) => {
 
     const connectedTestApps = allConnectedApps.filter((connectedApp: ConnectedApp) => connectedApp.is_test_app);
     const connectedApps = allConnectedApps.filter((connectedApp: ConnectedApp) => !connectedApp.is_test_app);
+    const pendingApps = allConnectedApps.filter((connectedApp: ConnectedApp) => connectedApp.is_pending);
```

Comment:
```suggestion
    const hasPendingApps = undefined !== allConnectedApps.find((connectedApp: ConnectedApp) => connectedApp.is_pending);
```
The intent is clearer IMO since pending apps aren't used in isolation